modules/cluster-logging-updating-logging.adoc

```diff
@@ -163,3 +185,22 @@ $ oc get CronJob curator -n openshift-logging -o yaml |grep 'image:'
 ----
 image: registry.redhat.io/openshift4/ose-logging-curator5@sha256:330c3499e790d0e184414125a4843cd48849c601eb9f19ff82f30794c858b0bc
 ----
+
+.Post install
+
+After Cluster Logging and Elasticsearch Operators are fully updated to 4.5, you must perform the following actions to complete the upgrade.
+
+* Recreate your Kibana index patterns. Because of changes in Kibana, the cluster logging upgrade does not automatically create index patterns. 
+
+** Use the following command to get a list of new indices in your cluster:
++
+----
+$ oc exec <elasticsearch-pod> -- es_util --query='_cat/indices'
+----
+
+** Users must manually create index patterns for their projects. Because of the new Elasticseach data model, indices that were prefixed *project-* in {product-title} 4.4 must be prefixed with *app-*. Create the new index patterns with the prefixe *app-* and use the `@timestamp` time field.
```

Comment:
This is not correct. Indices and indexes are not the same as index pattern.  Index patterns is a Kibana thing that is an abstraction to a set of indices.  User's need to create index patterns but do not need to do anything to indices.  Updating the verbiage but generally [1] is the script I used for CEE enablement which identifies what users need to do:

* Open Kibana
* Create index pattern "app"
* Create index pattern "infra" if you are an admin
* Create index pattern "audit if you are an admin
* Chouse @timestamp field for all of them

The text should probably be something like:

```
** Users must manually create index patterns to see logs  for their projects. Because of the new Elasticsearch data model, all logs that were stored in an index prefixed like *project-* in {product-title} 4.4 are now co-located in a single index prefixed with *app-*. User's should create a new index patterns named 'app" and use the `@timestamp` time field.
```

[1] Ref https://docs.google.com/document/d/1BbludxtoowKcgfSscH4fxmsXnX6NNRvyiG_Kdt7EA0A/edit?usp=sharing

Reply:
We may also want to make a statement there is no need to create an index pattern to a specific index or wildcarded index

Reply:
@jcantrill There is a single index Create index pattern "app"? Each user with a project needs to log in to Kibana and create an index pattern from the `app` index?

Reply:
@jcantrill @sichvoge ^^